pl-backend/Services/TokenService.cs

```diff
@@ -0,0 +1,103 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.IdentityModel.Tokens;
+using pl_backend.Models;
+using System.Data;
+using System.IdentityModel.Tokens.Jwt;
+using System.Net.Http.Headers;
+using System.Security.Claims;
+
+namespace pl_backend.Services
+{
+    public interface ITokenService
+    {
+        string CreateToken(User user);
+        Token? ValidateToken(string authorization);
+        User GetCurrentUser();
+    }
+    public class TokenService : ITokenService
+    {
+        private readonly IConfiguration _configuration;
+        private readonly IHttpContextAccessor _contextAccessor;
+
+        public TokenService(IConfiguration configuration)
+        {
+            _configuration = configuration;
+        }
```

Comment:
Missing dependency injection
```suggestion
        public TokenService(IConfiguration configuration, IHttpContextAccessor contextAccessor)
        {
            _configuration = configuration;
            _contextAccessor = contextAccessor;
        }
```